lib/dashboard/repl.js

```diff
@@ -0,0 +1,39 @@
+const repl = require("repl");
+
+const Console = require('./console.js');
+
+class REPL {
+  constructor(options) {
+    this.env = options.env;
+    this.plugins = options.plugins;
+    this.events = options.events;
+    this.console = new Console({
+      events: this.events,
+      plugins: this.plugins,
+      version: options.version
+    });
+  }
+
+  enhancedEval(cmd, context, filename, callback) {
+    this.console.executeCmd(cmd.trim(), (result) => {
```

Comment:
Without the trim any string comparison won't work, because each command ends with \n (it is invisible when doing an output)